src/main/java/com/mindee/documentparser/DocumentManipulation.java

```diff
@@ -0,0 +1,21 @@
+package com.mindee.documentparser;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class DocumentManipulation {
+
+  public enum DocumentManipulationMode{
+    KEEP_ONLY_LISTED_PAGES,
+    REMOVE_LISTED_PAGES
+  }
+
+  private DocumentManipulationMode mode;
+  private List<Integer> pages;
+  @Builder.Default
+  private Integer manipulateDocumentIfPagesGreaterThan = 10;
```

Comment:
``OnMinPages`` to be consistent with Python SDK, Node.js SDK and .NET SDK

Reply:
will refactor